pages/provinces/[provinceSlug]/index.tsx

```diff
@@ -110,6 +116,15 @@ export default function ProvincePage(props: ProvinceProps) {
             isLoading={isLoading}
             provinceSlug={provinceSlug}
           />
+          <SeoText
+            textNode={
+              <span>
+                Daftar Informasi Fasilitas Kesehatan (Faskes) & Alat Kesehatan
+                (Alkes) untuk COVID-19 di {provinceName} per {getCurrentDate()}{" "}
+                {getCurrentMonth()} {getCurrentYear()}.
+              </span>
```

Comment:
How about using `<p>` tag instead of `<span>`?

Reply:
Done